.devcontainer/post_start.sh

```diff
@@ -0,0 +1,55 @@
+#!/bin/bash
+
+# This script assumes we are running in a Miniconda container where:
+# - /opt/conda is the Miniconda installation directory
+# - https://github.com/conda/conda is mounted at /workspaces/conda
+# - https://github.com/conda/conda-libmamba-solver is mounted at
+#   /workspaces/conda-libmamba-solver
+
+set -euo pipefail
+
+MINICONDA=${MINICONDA:-/opt/conda}
+CONDA_SRC=${CONDA_SRC:-/workspaces/conda}
+CONDA_LIBMAMBA_SOLVER_SRC=${CONDA_LIBMAMBA_SOLVER_SRC:-/workspaces/conda-libmamba-solver}
+MAMBA_SRC=${MAMBA_SRC:-/workspaces/mamba}
+
+function recompile-mamba () {
+  if [ -f "$MAMBA_SRC/mamba/setup.py" ]; then
+    cd "$MAMBA_SRC"
+    sudo "$MINICONDA/bin/cmake" -B build/ \
+        -DBUILD_LIBMAMBA=ON \
+        -DBUILD_SHARED=ON \
+        -DCMAKE_INSTALL_PREFIX="$MINICONDA" \
+        -DCMAKE_PREFIX_PATH="$MINICONDA" \
+        -DBUILD_LIBMAMBAPY=ON
+    sudo "$MINICONDA/bin/cmake" --build build/ -j2
+    sudo make install -C build/
+    cd -
+  else
+    echo "ERROR: path '$MAMBA_SRC' not mounted so no libmamba to recompile"
+  fi
+}
+
+# We only support mamba 1.x for now; v2 there's no 'mamba' subdirectory
+if [ -f "$MAMBA_SRC/mamba/setup.py" ]; then
+  # remove "sel(win)" in environment yaml hack since conda does not understand
+  # libmamba specific specs
+  sed '/sel(.*)/d' "$MAMBA_SRC/mamba/environment-dev.yml" > /tmp/mamba-environment-dev.yml
+  sudo "$MINICONDA/condabin/conda" env update -p "$MINICONDA" \
+       --file /tmp/mamba-environment-dev.yml
```

Comment:
As is, this line gives me
```
Start: Run in container: /bin/sh -c bash /workspaces/conda-libmamba-solver/.devcontainer/post_start.sh
/workspaces/conda-libmamba-solver/.devcontainer/post_start.sh: line 38: sudo: command not found
```
We could just install sudo, but I think a (brief) discussion is needed on whether conda should be installed as `root` (as it is now), in which case no sudo should be needed, or as an ordinary user.

Reply:
Removed sudo usage for now.